elements/campaign-display/campaign-display.js

```diff
@@ -23,7 +23,7 @@ class CampaignDisplay extends BulbsElement {
   render () {
     return (
       <CampaignDisplayRoot
-          data={this.state}
+          {...this.state}
```

Comment:
With the way it is now, I have to go to another file to figure out what would potentially be passed in here, and that's with prior knowledge of what the structure here is.

If we want to reduce cognitive load, we should be explicit about what goes into this component.